workflow/rules/peddy.smk

```diff
@@ -0,0 +1,46 @@
+__author__ = "Padraic Corcoran"
+__copyright__ = "Copyright 2022, Padraic Corcoran"
+__email__ = "padraic.corcoran@scilifelab.uu.se"
+__license__ = "GPL-3"
+
+
+rule peddy:
+    input:
+        vcf="qc/peddy/all.vcf.gz",
+        ped="qc/peddy/all.ped",
+    output:
+        ped=temp("qc/peddy/peddy.peddy.ped"),
+        ped_check=temp("qc/peddy/peddy.ped_check.csv"),
+        sex_check=temp("qc/peddy/peddy.sex_check.csv"),
+        het_check=temp("qc/peddy/peddy.het_check.csv"),
+        ped_html=temp("qc/peddy/peddy.html"),
+        ped_vs_html=temp("qc/peddy/peddy.vs.html"),
+        pca=temp("qc/peddy/peddy.background_pca.json")
+    params:
+        pre="qc/peddy/peddy",
+        extra=config.get("peddy", {}).get("extra", "")
+    log:
+        "qc/peddy/peddy.output.log",
+    benchmark:
+        repeat(
+            "qc/peddy/all.peddy.benchmark.tsv",
+            config.get("peddy", {}).get("benchmark_repeats", 1),)
```

Comment:
Threads needed or at least we always use it.
threads: config.get("peddy", {}).get("threads", config["default_resources"]["threads"])